x/distribution/types/store.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// historical rewards for a validator
+// TODO add reference counter, ref https://github.com/cosmos/cosmos-sdk/pull/3099#discussion_r245747051
+// height is implicit within the store key
+type ValidatorHistoricalRewards = sdk.DecCoins
+
+// current rewards for a validator
```

Comment:
more in depth descriptions would be valuable for these new types

Reply:
Added some more in-depth descriptions, let me know if they help!

Reply:
they do thank you!